app.py

```diff
@@ -88,3 +91,13 @@ def get_all_applicants(listing_id):
     """Gets all applicants from <listing_id>"""
     data = db.get_applicants(table_name="zap-applications", listing_id=listing_id)
     return data
+
+
+@app.route("/listings/{id}/toggle/visibility", methods=["PATCH"], cors=True)
+def toggle_visibility(id):
+    """Toggles visibilility of a given <listing_id>"""
+    data = db.toggle_visibility(table_name="zap-listings", key={"listingId": id})
+    if data:
+        return {"status": True}
+
+    return {"status": False}
```

Comment:
Not technically a silent error, but it would be good to report that the call failed _because_ the listing was not found.

Reply:
Good catch - I'll make the changes 👍